airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java

```diff
@@ -181,92 +162,24 @@ public void run(final ConnectionUpdaterInput connectionUpdaterInput) throws Retr
 
   private CancellationScope generateSyncWorkflowRunnable(final ConnectionUpdaterInput connectionUpdaterInput) {
```

Comment:
can we leave a comment that it is intentional for all these to happen in this cancellation block since there are operations we want to cancel if someone hits cancels the sync job.

not clear otherwise.